BaSD-2021/German-MindSet-app
# BaSD-MS-17: Checkbox shared component

## Description:
Creation of a new Checkbox component. 
It is implemented in the forms corresponding to Postulants, Psychologists and Positions.
### Steps to test Postulant's Checkbox:
1. Run `npm run start` on both front and back-end apps.
2. Search for **Postulants** option on nav bar and click on it.
3. Click **add** or **edit** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Steps to test Psychologists's Checkbox:
1. Repeat step one.
2. Search for **Psychologists** option on nav bar and click on it.
3. Click **add** or **update** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Steps to test Positions's Checkbox:
1. Repeat step one.
2. Search for **Positions** option on nav bar and click on it.
3. Click **add** or **update** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Reviewers:
- @estefrare
- @nicolasblasco 
- @GuidoGlielmi 
- @ValentinaCollareta
